models/core/staging/core__stg_claims_procedure.sql

```diff
@@ -12,8 +12,7 @@ with unpivot_cte as (
 
 select
     claim_id as claim_id
-   ,claim_line_number as claim_line_number
```

Comment:
What is the reason/benefit of changing the name of this field to `procedure_sequence_id` and then manually listing out the sequence value (1-25) in the unioned tables? The `claim_line_number` is part of the primary key for a claim, so I'm not sure it's a good idea to remove it.

Reply:
`claim_line_number` was just added last week and is not used in or part of the final output. It serves no purpose as far as i can tell.
The reason I am adding a procedure sequence id is there is a uniqueness criteria on the `procedure_id` field, and since it is possible to have the same procedure more than once in a claim, i needed to add a proc seq id to the `procedure_id` field. For the hcpcs/cpt procedures it makes sense to use claim line, but for the institutional procs 1-25 it makes sense to use the seq numbers from the column names. These institutional procedures are usually duplicated across the claim lines

Reply:
@sarah-tuva `procedure_id` is defined as
```
        "unpivot_cte.data_source",
        "'_'",
        "unpivot_cte.claim_id",
        "'_'",
        "unpivot_cte.source_code",
        "case when unpivot_cte.modifier_1 is not null then CONCAT('_', unpivot_cte.modifier_1) else '' end",
        "case when unpivot_cte.modifier_2 is not null then CONCAT('_', unpivot_cte.modifier_2) else '' end",
        "case when unpivot_cte.modifier_3 is not null then CONCAT('_', unpivot_cte.modifier_3) else '' end",
        "case when unpivot_cte.modifier_4 is not null then CONCAT('_', unpivot_cte.modifier_4) else '' end",
        "case when unpivot_cte.modifier_5 is not null then CONCAT('_', unpivot_cte.modifier_5) else '' end",
        "case when unpivot_cte.practitioner_npi is not null then CONCAT('_', unpivot_cte.practitioner_npi) else '' end"
```
which isn't granular enough to capture institutional procedures, when you want to report out on information like date and provider. I think the inclusion of procedure sequence number is necessary as part of the id

Reply:
Thank you for the details. I'm going to have a couple of other folks take a look to make sure we're not missing something.